types/Number.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const GraphQL = require('graphql');
+
+module.exports = (field) => ({
+	type: GraphQL.GraphQLInt,
```

Comment:
I am not sure that the number field is limited to integers. Please confirm and, if it works with floats as well, update this type.